### DISCUSSION TITLE
how to get certified

### DISCUSSION THREAD

**Discussion by ShindeSharayu24** [Cyfrin/foundry-full-course-cu#1794] [Q&A]
I am about to complete the Blockchain (2024)  full course , but from where can i get it certified without having to enroll for any courses , i need to get it certified for college and  LinkedIn purposes

**Comment by EngrPips** [comment]
Hello @clumsycurlys, I believe there is something about the certificate in the Cyfrin pipeline. Still, before Cyfrin officially hands out the certificate, you can update your LinkedIn profile to reflect that you took and completed the Cyfrin updraft course.
